lib/import/colorectal/core/colorectal_handler_mapping.rb

```diff
@@ -4,7 +4,8 @@ module Core
       # Provides the handler appropriate for the dataformat from each center
       class ColorectalHandlerMapping
         HANDLER_MAPPING = {
-          'RR8' => Import::Colorectal::Providers::Leeds::LeedsHandlerColorectal,
+          'RR8_2' => Import::Colorectal::Providers::Leeds::LeedsHandlerColorectal,
+          'RR8' => Import::Colorectal::Providers::Leeds::LeedsHandlerNewColorectal,
```

Comment:
Similarly, I'm not sure about the "New" naming convention, what would happen if there is a third one, for example? Could the naming be versioned?